src/OrchardCore.Modules/OrchardCore.Workflows/Views/WorkflowTrimming.Fields.Edit.cshtml

```diff
@@ -0,0 +1,111 @@
+@using OrchardCore.Workflows.Models
+@using OrchardCore.Workflows.Trimming.Services
+@using OrchardCore.Workflows.Trimming.ViewModels
+@model WorkflowTrimmingViewModel
+
+<div class="form-check">
```

Comment:
usually this is wrapped with `<div class="mb-3"></div>`. Did it look better without the margin?

Reply:
This is not something I've written either, just the diff view shows this as a new file despite only being renamed. But I don't suppose much thought went into this, and with `mb-3` looks better.